app/models/form/lettings/pages/max_rent_value_check.rb

```diff
@@ -12,21 +12,12 @@ def initialize(id, hsh, subsection, check_answers_card_number: nil)
         },
       ],
     }
-    @informative_text = {
-      "translation" => "soft_validations.rent.max_hint_text",
-      "arguments" => [
-        {
-          "key" => "field_formatted_as_currency",
-          "arguments_for_key" => "soft_max_for_period",
-          "i18n_template" => "soft_max_for_period",
-        },
-      ],
-    }
+    @informative_text = {}
```

Comment:
I think you can delete this line if there is no informative text

Reply:
I did originally, but I had to add it back as it was failing tests, I looked around and there are other instances of a blank informative_text field and I presume they ran into the same problem.